tlatools/org.lamport.tlatools/src/tlc2/util/BufferedRandomAccessFile.tla

```diff
@@ -0,0 +1,384 @@
+\* Copyright (c) 2024, Oracle and/or its affiliates.
+
+----------------------- MODULE BufferedRandomAccessFile -----------------------
```

Comment:
Just wanted to say that I have absolutely found it useful to have a spec of the class for review, and especially as a refinement.

Reply:
This spec could also be referenced at tlaplus/examples